public/app/percona/dbaas/components/DBCluster/EditDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -93,14 +121,26 @@ export const DBClusterAdvancedOptions: FC<DBClusterAdvancedOptionsProps> = ({ se
 
   useEffect(() => {
     if (selectedCluster) {
-      getResources();
+      getAllocatedResources();
 
-      timer = setInterval(() => getResources(false), RECHECK_INTERVAL);
+      allocatedTimer = setInterval(() => getAllocatedResources(false), RECHECK_INTERVAL);
```

Comment:
consider using a recursive `setTimeout` instead, to avoid chances of overlapping calls:

```js
let timerId = setTimeout(function getResources() {
  getAllocatedResources(false)
  timerId = setTimeout(getResources, RECHECK_INTERVAL);
}, RECHECK_INTERVAL);
```

this is considered a more robust solution. Try to avoid `setInterval`.

Reply:
This will have to be changed in PMM-7674 to wait for the previous request anyway, so no need to do it here.